src/cases/containers/case-hearings/case-hearings.component.html

```diff
@@ -6,7 +6,7 @@
      class="govuk-button
      govuk-button--start"
      data-module="govuk-button"
-     [routerLink]="['/', 'hearings', 'request']"
+     [routerLink]="['/', 'hearings', 'request', 'locationSearch']"
```

Comment:
Can we revert this and ask QA to test on:
/hearings/request/locationSearch url directly?